libs/payments/stripe/src/lib/stripe.client.ts

```diff
@@ -3,8 +3,22 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { Injectable } from '@nestjs/common';
+import { Stripe } from 'stripe';
+
+import { StripeClientConfig } from './stripe.client.config';
 
 @Injectable()
 export class StripeClient {
-  constructor() {}
+  public readonly stripe: Stripe;
+
+  constructor(private stripeClientConfig: StripeClientConfig) {
+    this.stripe = new Stripe(this.stripeClientConfig.apiKey, {
+      apiVersion: '2022-11-15',
+      maxNetworkRetries: 3,
+    });
+  }
+
+  async finalizeInvoice(invoiceId: string, opts?: any) {
```

Comment:
Opts should be typed. You can grab the typings from Stripe's `finalizeInvoice` via `Parameters<typeof this.stripe.invoices.finalizeInvoice>`